### DISCUSSION TITLE
Is there any way to search within Documentation?

### DISCUSSION THREAD

**Discussion by sOMarkus** [uikit/uikit#4658] [Q&A] [answered]
Hi,
is there any way to search within [Documentation hosted at getuikit.com](https://getuikit.com/docs/introduction)?
I'd start working with UIkit for new project in our company and really like it - except the fact that I sadly can't really search within documentation it seems :(

Anybody know if it's possible or maybe a feature planned for future? Thanks!

**Comment by BernhardBaumrock** [comment]
Unfortunately there is no good search that I know of.

I'm using a custom google chrome search, so I can type `ui flex center` and it will take me to this page: https://www.google.com/search?q=site%3Agetuikit.com+flex+center

@uikit team
Have a look at https://tailwindcss.com/docs/functions-and-directives how the search is implemented - that's really a huge step forward!

Algolia offers its product for free for open source: https://www.algolia.com/for-open-source/

**Reply by sOMarkus** [reply]
@BernhardBaumrock
Thanks. My Workaround is close to yours.
If I do not find something directly in the Index I'll just type into google search within Edge and get at least the location within documentation.

Tailwindcss search looks really cool and is easy to use. Thats really something I'd like to see for uikit documentation.

**Comment by sOMarkus** [comment, marked as answer]
New available search by Algolia within Docu is really divine! 
![image](https://github.com/uikit/uikit/assets/71393421/8bcf4c4a-107d-4d45-b618-94c6556793a4)

**Reply by BernhardBaumrock** [reply]
Did you implement that and create a PR?

**Reply by BernhardBaumrock** [reply]
Oh ... It's already LIVE!!!

**Reply by sOMarkus** [reply]
> Did you implement that and create a PR?

No credits to me - but I'm glad it found a way inside the documentation.
Just want to note it down here as I'd see it's available now and close the discussion.

**Reply by BernhardBaumrock** [reply]
Nice, thx for the update :)
